README.md

```diff
@@ -49,3 +49,35 @@ Create a kubernetes manifest for a pod which will containa ToDo app container:
 1. `README.md` Should have explained your strategy configuration (Why such numbers)
 1. `README.md` Should have explained how to access the app after deployment
 1. Create PR with your changes and attach it for validation on a platform.
+
+
+In order to deploy the app to k8s you should execute the next commands:
+kubectl apply -f deployment.yml
+kubectl apply -f hpa.yml
```

Comment:
What about namespace?

Reply:
fixed
Needed description was added to the readme file